src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[]{
+            new FigureSupplier().getRandomFigure(),
+            new FigureSupplier().getRandomFigure(),
+            new FigureSupplier().getRandomFigure(),
+            new FigureSupplier().getDefaultFigure(),
+            new FigureSupplier().getDefaultFigure(),
+            new FigureSupplier().getDefaultFigure()

```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. Let's do it only once - before the loop starts. This will help to avoid unnecessary object creation and make the code cleaner.